### ISSUE TITLE
Task 1: Java Installation 🌟

### ISSUE THREAD

**Issue by here-tunan** [The-Learners-Circle/our-blog#3]
Hi, guys. Now, our readme lacks a detailed Java installation process. Can anyone try to install it and complete the tutorial?
Please specify the machine system on which you have installed Java.

**Comment by KingJoFr** [comment]
I could try this, but I use ubuntu.  It might not be very helpful. Does anyone else use a linux operating system?

**Comment by here-tunan** [comment]
Yes, I think you could write the installation process in ubuntu.
# Java installation
xxxxxx
## ubuntu(Linux) installation
xxxxxxx
write here
xxxxxxx
## macOS
## Windows
